Cargo.toml

```diff
@@ -8,3 +8,5 @@ edition = "2018"
 [dependencies]
 bitcoin = "0.23.0"
 bitcoin_hashes = "0.7.6"
+sha2 = "0.10.2"
+serde_json="1.0.81"
```

Comment:
We could have this under `[dev-dependencies]` instead of `[dependencies]`

Reply:
Agree for `serde_json`. `sha2` can't be dev dependence because it's used on main code. `bitcoin-hashes` doesn't support sha512_256 as used for utreexo (it took me a little while to figure that out).